experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -157,14 +157,12 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 		 * This is used rather than the Fluid client ID because the Fluid client ID is not stable across reconnections.
 		 */
 		const localSessionId = uuid();
-		const undoRedoManager = new UndoRedoManager(repairDataStoreProvider, changeFamily, () =>
-			this.editManager.getLocalBranchHead(),
-		);
+		const undoRedoManager = new UndoRedoManager(repairDataStoreProvider, changeFamily);
```

Comment:
Since you've removed the Kreepy Klosures, you can get rid of all those comments in the EditManager constructor about "You're not allowed to call getHead". I think there are some in another module too? Was that SharedTreeBranch?